app/src/main/java/campus/tech/kakao/contacts/ContactListAdapter.kt

```diff
@@ -0,0 +1,52 @@
+package campus.tech.kakao.contacts
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import campus.tech.kakao.contacts.model.Contact
+
+class ContactListAdapter(
+    private val inflater: LayoutInflater,
+    val context: Context,
+    private val contactList: MutableList<Contact>,
```

Comment:
여기서 사용하는 ContactList는 Mutable 하지 않아도 되겠네요.
데이터를 여러 객체가 서로 공유할 때 수정 및 변경은 최대한 하나의 객체가 도맡는게 좋습니다.
민철님의 경우 이런 책임을 Activity가 갖는 것으로 보이네요. 그러면 ContactListAdapter는 캐싱의 용도로만 사용하는게 좋아보입니다.
```suggestion
    private val contactList: List<Contact>,
```

Reply:
수정하였습니다!